### PR TITLE
Add support for Django 4.X

### PR DESCRIPTION
**Pull request by jpintor52**
### Overview
Add support for Django `4.0`, `4.1`, and `4.2`

### How to test
- [ ] Test pass

### CODE REVIEW

**Review by nichaydel — COMMENTED** [review]
Looks good, just had a few questions before approving

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Core packages needed
-Django>=3.2, <4.0
+Django>=3.2, <6.0
```
Comment (nichaydel) [inline requirements.txt:2]: Is there a reason this is 6.0 as opposed to the highest supported version?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -30,8 +30,11 @@ jobs:
           --health-retries 5
     strategy:
       matrix:
-        python-version: [ "3.7", "3.8", "3.9" ]
-        django-version: [ "3.2" ]
+        python-version: [ "3.8", "3.9", "3.10"]
+        django-version: [ "3.2" , "4.0", "4.1", "4.2"]
+        include:
+          - python-version: "3.10"
+            django-version: "5.0"
```
[the remark this answers is not in the document]
Comment (jpintor52) [reply .github/workflows/main.yml:37]: Django 5.0 only supports Python 3.10 and above. That's why I added it as an `include`

File: requirements.txt (hunk 1 above)
Comment (jpintor52) [reply requirements.txt:2]: The highest version currently is 5.0. I want to make the package ready for the 5.0 upgrade, so the next person that needs to update Django doesn't have any issues.

**Review by nichaydel — APPROVED** [review]
LGTM

**Pull request merged**